manifests/init.pp

```diff
@@ -19,6 +19,9 @@
 # $proxy_password::     Proxy password for authentication
 #                       type:String
 #
+# $pulp_max_speed::     Maximum bandwidth used per download in bytes/sec from CDN
+#                       Type:Integer[Optional]
```

Comment:
Not if its going to use https://github.com/Katello/puppet-pulp/pull/189#issuecomment-273553092 ;)